### ISSUE TITLE
Calculating results server-side

### ISSUE THREAD

**Issue by unpairedbracket** [jstockwin/Film-Night#64]
I've been looking into node.js and it turns out we wouldn't actually need to use it in its webserver-type capacity to use Tom's js on the server-side. As long as we've got node installed on the pis, it might be as easy as writing a bit of js and calling node from exec in php to run the functions. I'll look more into this later.

**Comment by jstockwin** [comment]
I agree that this is possible, but am not sure it's really worth it. 

It would allow us to auto-remove the winning film from the film list, but the winning film is not always watched in reality, so this is potentially not really worth it?
